resdk/tests/unit/test_resolwe.py

```diff
@@ -326,7 +331,7 @@ def test_call_with_all_args(self, resolwe_mock, os_mock, data_mock):
 class TestUploadFile(unittest.TestCase):
 
     def setUp(self):
-        self.fn = os.path.normpath(os.path.dirname(__file__) + "./../files/example.fastq")
+        self.file_path = os.path.normpath(os.path.dirname(__file__) + "./../files/example.fastq")
```

Comment:
Is there a reason to prefer `os.path.normpath` over more frequently used `os.path.abspath`?

Also the `./` part in the file name is redundant?